proc_linewidth.py

```diff
@@ -9,57 +9,71 @@
 from symfit.contrib.interactive_guess import InteractiveGuess
 import numpy as np
 
-fl = fl_mod()
-for searchstr,exp_type,postproc in [
-        ["201118_1mM4AT",'ESR','ESR_linewidth']
+B_center = Parameter('B_center', value=-0.2)
+sigma = Parameter('sigma', value=3)
+R = Parameter('R', value=7.0)
+A = Parameter('A', value=3e4)
+C = Parameter('C', value=0)
+B = Variable('B')
+y_var = Variable('y')
+conc_list = []
+R_list = []
+sigma_list = []
+for searchstr,exp_type,postproc,thisguess,interactive,concentration in [
+        ("201118_10mM4AT",'francklab_esr/alex','ESR_linewidth',
+            { # here, I entered based on the next, and then copied and pasted the result
+                A: 952.059548814867,
+                B_center: 0.645245462402653,
+                R: 1.1452099193249357,
+                sigma: 0.7520327837593248,
+                },
+            False,
+            10e-3
+            ),
+        ("201118_1mM4AT",'francklab_esr/alex','ESR_linewidth',
+            {
+
+                A:         4e2,
+                B_center:  4.220642e-01,
+                R:         3.617994e-01,
+                sigma:     8.702270e-01,
+                },
+            False,
+            1e-3
+            )
         ]:
     d = find_file(searchstr + '.DSC', exp_type=exp_type, postproc=postproc,
                   lookup=postproc_dict)
-    fl.next('linewidth for 10mM 4AT')
-    fl.plot(d)
+    plt.figure()
+    plt.title('linewidth for 10mM 4AT')
```

Comment:
why not use fl.plot? why use the matplotlib stuff?

Reply:
Because the interactive plot is as already directly accessing matplotlib features